site/public/js/forum.js

```diff
@@ -1217,6 +1217,45 @@ function alterShowMergeThreadStatus(newStatus, course) {
     location.reload();
 }
 
+// eslint-disable-next-line no-unused-vars
+function markUnread(current_thread, current_user) {
+    const url = buildCourseUrl(['forum', 'threads', 'mark_unread']);
```

Comment:
Shouldn't the url also include thread_id and afterwards mark_unread. 
Example:
![image](https://github.com/Submitty/Submitty/assets/70663754/ec3515bb-35ae-48ad-96c6-304aea90529d)